frontend-react/src/resources/PermissionsResource.ts

```diff
@@ -1,6 +1,8 @@
-export const PERMISSIONS = {
-    'sender': 'DHSender',
-    // for the permission check on the /daily page - there is no corresponding Okta group/permission.
-    // This is purely for the front-end
-    'receiver': 'DHReceiver'
+export enum PERMISSIONS {
+    /* Non-Okta pseudo groups */
+    SENDER = "DHSender",
+    RECEIVER = "DHReceiver",
+    
+    /* Okta groups */
+    PRIME_ADMIN = "DHPrimeAdmins"
```

Comment:
Looks like `frontend/src/js/prime-web-receiver-pre.js` (non-react) has this string hard coded in a few places, it's nice to have it defined in a single place. Thanks!

Reply:
No problem! If you see any code that can be altered to use these instead of strings on the React side, feel free to patch that up :smiley: I didn't check _every_ file, but I got the main ones from `App.ts`